doc/d-practical-exercises/3-invkine.md

```diff
@@ -21,28 +21,35 @@ the tool. They are displayed when the robot moves.
 Example of how to use the robot is has below.
 
 ```py
-from os.path import join
-import pinocchio as se3
-from mobilerobot import MobileRobotWrapper
+import pinocchio as pin
 from pinocchio.utils import *
-
-PKG = '/opt/openrobots/share'
-URDF = join(PKG, 'ur5_description/urdf/ur5_gripper.urdf')
-
-robot = MobileRobotWrapper(URDF, [PKG])
+from os.path import dirname, join, abspath
+from mobilerobot import MobileRobotWrapper
+import time
+# Starting gepetto server and give a time
+import gepetto.corbaserver
+
+gepetto.corbaserver.start_server()
+time.sleep(5)
+
+pinocchio_model_dir = join(dirname(dirname(str(abspath(__file__)))), "models")
+model_path = join(pinocchio_model_dir, "example-robot-data/robots/ur_description")
+mesh_dir = pinocchio_model_dir
+urdf_filename = "ur5_gripper.urdf"
+urdf_model_path = join(join(model_path, "urdf"), urdf_filename)
+robot = MobileRobotWrapper(urdf_model_path, pinocchio_model_dir)
 robot.initDisplay(loadModel=True)
 
-# robot.viewer.gui.addFloor('world/floor')
-
 NQ, NV = robot.model.nq, robot.model.nv
 
-q = rand(NQ)
-robot.display(q)
+# create valid random position
+q = pin.integrate(robot.model, pin.neutral(robot.model), np.random.rand(NV) * 10)
+robot.viz.display(q)
```

Comment:
just robot.display should be fine, you don't need robot.viz here, or in other places in this PR.